engine/src/main/java/org/terasology/rendering/dag/stateChanges/DisableDepthWriting.java

```diff
@@ -38,6 +38,7 @@
      * two task instances frame the execution of a node's process() method unless they are deemed redundant,
      * i.e. because the upstream or downstream node also disables depth buffer writing.
      */
+    // TODO: Remove this method?
```

Comment:
It does look unnecessary. Perhaps it's something that made sense in earlier commits and just tagged along, unnoticed. Let's try and remove it.